internal/server/mock/cron/server.go

```diff
@@ -0,0 +1,131 @@
+package cron
+
+import (
+	"context"
+	"time"
+
+	"github.com/skpr/api/pb"
+)
+
+var state = map[string]*pb.CronDetail{
```

Comment:
We could move the state to an "environment" state that has the lot eg. config, cron, database images etc